### ISSUE TITLE
gst-plugins-good-1.20.1 fails to build

### ISSUE THREAD

**Issue by klemensn** [NixOS/nixpkgs#176300]
### Steps To Reproduce
`nixos-rebuild test -I nixpkgs=/path/to/nixpkgs` at 236cc2971ac72acd90f0ae3a797f9f83098b17ec

### Build log
```
building '/nix/store/7vbzb4vyhl97m1mbn4j3nz5q15656v95-gst-plugins-good-1.20.1.drv'...
qtPreHook
unpacking sources
unpacking source archive /nix/store/x1z59xzyp4jw7821gn90jxjid9frvd88-gst-plugins-good-1.20.1.tar.xz
source root is gst-plugins-good-1.20.1
setting SOURCE_DATE_EPOCH to timestamp 1647257865 of file gst-plugins-good-1.20.1/po/gst-plugins-good-1.0.pot
patching sources
patching script interpreter paths in scripts/extract-release-date-from-doap-file.py
scripts/extract-release-date-from-doap-file.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/553d7c4xcwp9j1a1gb9cb1s9ry3x1pi9-python3-3.9.13/bin/python3"
configuring
meson flags: --buildtype=plain         --libdir=/nix/store/pa8jyip3v45zl3rv62kn5gwwq2hdyx25-gst-plugins-good-1.20.1/lib --libexecdir=/nix/store/pa8jyip3v45zl3rv62kn5gwwq2hdyx25-gst-plugins-good-1.20.1/libexec         --bindir=/nix/store/pa8jyip3v45zl3rv62kn5gwwq2hdyx25-gst-plugins-good-1.20.1/bin --sbindir=/nix/store/pa8jyip3v45zl3rv62kn5gwwq2hdyx25-gst-plugins-good-1.20.1/sbin         --includedir=/nix/store/2k9wc4scklni0iwysdlslpi0p56vm86g-gst-plugins-good-1.20.1-dev/include         --mandir=/nix/store/pa8jyip3v45zl3rv62kn5gwwq2hdyx25-gst-plugins-good-1.20.1/share/man --infodir=/nix/store/pa8jyip3v45zl3rv62kn5gwwq2hdyx25-gst-plugins-good-1.20.1/share/info         --localedir=/nix/store/pa8jyip3v45zl3rv62kn5gwwq2hdyx25-gst-plugins-good-1.20.1/share/locale         -Dauto_features=enabled         -Dwrap_mode=nodownload         --prefix=/nix/store/pa8jyip3v45zl3rv62kn5gwwq2hdyx25-gst-plugins-good-1.20.1 -Dexamples=disabled -Ddoc=disabled -Dglib-asserts=disabled -Dgtk3=disabled -Drpicamsrc=disabled 
The Meson build system
Version: 0.61.2
Source dir: /build/gst-plugins-good-1.20.1
Build dir: /build/gst-plugins-good-1.20.1/build
Build type: native build
Project name: gst-plugins-good
Project version: 1.20.1
building '/nix/store/3x76dysb874z3q9ky1s9c4p3p26hzwxs-nixos-version_fish-completions.drv'...
building '/nix/store/d2ivzizd428qw5a5389fz2cdv5lzrz43-options.json.drv'...
C compiler for the host machine: gcc (gcc 11.3.0 "gcc (GCC) 11.3.0")
C linker for the host machine: gcc ld.bfd 2.38
Host machine cpu family: x86_64
Host machine cpu: x86_64
C++ compiler for the host machine: g++ (gcc 11.3.0 "g++ (GCC) 11.3.0")
C++ linker for the host machine: g++ ld.bfd 2.38
Compiler for C supports link arguments -Wl,-Bsymbolic-functions: YES 
Compiler for C supports arguments -fvisibility=hidden: YES 
Compiler for C supports arguments -fno-strict-aliasing: YES 
Message: Disabling GLib asserts
Has header "dlfcn.h" : YES 
Has header "fcntl.h" : YES 
Has header "inttypes.h" : YES 
Has header "memory.h" : YES 
Has header "process.h" : NO 
Has header "stdint.h" : YES 
Has header "stdlib.h" : YES 
Has header "strings.h" : YES 
Has header "string.h" : YES 
Has header "sys/ioctl.h" : YES 
Has header "sys/param.h" : YES 
Has header "sys/socket.h" : YES 
Has header "sys/stat.h" : YES 
Has header "sys/time.h" : YES 
Has header "sys/types.h" : YES 
building '/nix/store/l81hn1mza46cn749j7p3yiwkb30q5fl9-unit-script-nix-gc-start.drv'...
Has header "unistd.h" : YES 
Library m found: YES
building '/nix/store/zykm7rhc1zz8cwzly6mk0r9fd74dxr7w-options.xml.drv'...
Checking for function "asinh" with dependency -lm: YES 
Checking for function "clock_gettime" with dependency -lm: YES 
Checking for function "cosh" with dependency -lm: YES 
Checking for function "dcgettext" with dependency -lm: YES 
Checking for function "getpagesize" with dependency -lm: YES 
Checking for function "isinf" with dependency -lm: YES 
Checking for function "mmap" with dependency -lm: YES 
Checking for function "mmap64" with dependency -lm: NO 
Checking for function "sinh" with dependency -lm: YES 
Checking for function "gmtime_r" with dependency -lm: YES 
Checking for size of "char" : 1
Checking for size of "int" : 4
Checking for size of "long" : 8
Checking for size of "short" : 2
Checking for size of "void*" : 8
Checking for size of "off_t" : 8
Header <dlfcn.h> has symbol "RTLD_NOLOAD" : YES 
Compiler for C supports arguments -Wmissing-declarations: YES 
Compiler for C++ supports arguments -Wmissing-declarations: YES 
Compiler for C supports arguments -Wredundant-decls: YES 
Compiler for C++ supports arguments -Wredundant-decls: YES 
Compiler for C supports arguments -Wwrite-strings: YES 
Compiler for C++ supports arguments -Wwrite-strings: YES 
Compiler for C supports arguments -Winit-self: YES 
Compiler for C++ supports arguments -Winit-self: YES 
Compiler for C supports arguments -Wmissing-include-dirs: YES 
Compiler for C++ supports arguments -Wmissing-include-dirs: YES 
Compiler for C supports arguments -Wno-multichar: YES 
Compiler for C++ supports arguments -Wno-multichar: YES 
Compiler for C supports arguments -Wvla: YES 
Compiler for C++ supports arguments -Wvla: YES 
Compiler for C supports arguments -Wpointer-arith: YES 
Compiler for C++ supports arguments -Wpointer-arith: YES 
Compiler for C supports arguments -Wmissing-prototypes: YES 
Compiler for C supports arguments -Wdeclaration-after-statement: YES 
Compiler for C supports arguments -Wold-style-definition: YES 
Compiler for C supports arguments -Waggregate-return: YES 
Found pkg-config: /nix/store/0sjmycy1s616jag5fvsczjgzj7rf0kc2-pkg-config-wrapper-0.29.2/bin/pkg-config (0.29.2)
Run-time dependency gstreamer-1.0 found: YES 1.20.1
Run-time dependency gstreamer-base-1.0 found: YES 1.20.1
Run-time dependency gstreamer-net-1.0 found: YES 1.20.1
Run-time dependency gstreamer-controller-1.0 found: YES 1.20.1
Run-time dependency gstreamer-check-1.0 found: YES 1.20.1
Run-time dependency gstreamer-pbutils-1.0 found: YES 1.20.1
Run-time dependency gstreamer-allocators-1.0 found: YES 1.20.1
Run-time dependency gstreamer-app-1.0 found: YES 1.20.1
Run-time dependency gstreamer-audio-1.0 found: YES 1.20.1
Run-time dependency gstreamer-fft-1.0 found: YES 1.20.1
Run-time dependency gstreamer-riff-1.0 found: YES 1.20.1
Run-time dependency gstreamer-rtp-1.0 found: YES 1.20.1
Run-time dependency gstreamer-rtsp-1.0 found: YES 1.20.1
Run-time dependency gstreamer-sdp-1.0 found: YES 1.20.1
Run-time dependency gstreamer-tag-1.0 found: YES 1.20.1
Run-time dependency gstreamer-video-1.0 found: YES 1.20.1
Run-time dependency gstreamer-gl-1.0 found: YES 1.20.1
Message: GStreamer OpenGL window systems: wayland x11
Message: GStreamer OpenGL platforms: egl glx
Message: GStreamer OpenGL apis: gles2 gl
Run-time dependency gstreamer-gl-prototypes-1.0 found: YES 1.20.1
Run-time dependency gstreamer-gl-x11-1.0 found: YES 1.20.1
Run-time dependency gstreamer-gl-wayland-1.0 found: YES 1.20.1
Run-time dependency gstreamer-gl-egl-1.0 found: YES 1.20.1
Run-time dependency zlib found: YES 1.2.12
Run-time dependency glib-2.0 found: YES 2.72.1
Run-time dependency gobject-2.0 found: YES 2.72.1
Run-time dependency gio-2.0 found: YES 2.72.1
Run-time dependency orc-0.4 found: YES 0.4.32
Program orcc found: YES (/nix/store/8ahhb6519qa0ryjdcylq8jj5bjaghlrf-orc-0.4.32-dev/bin/orcc)
Fetching value of define "__ILP32__" :  
Program nasm found: YES (/nix/store/70ij88lq47r2vvag86jk07nwifna5rd2-nasm-2.15.05/bin/nasm)
Message: nasm found on x86-64
Header <gst/gstconfig.h> has symbol "GST_DISABLE_GST_DEBUG" with dependency gstreamer-1.0: NO 
Message: GStreamer debug system is enabled
Program python3 found: YES (/nix/store/553d7c4xcwp9j1a1gb9cb1s9ry3x1pi9-python3-3.9.13/bin/python3.9)
WARNING: Python files installed by Meson might not be found by python interpreter.
 This warning can be avoided by setting "python.platlibdir" option.
WARNING: Python files installed by Meson might not be found by python interpreter.
 This warning can be avoided by setting "python.purelibdir" option.
Message: Nasm configured on x86-64
Library bz2 found: YES
Has header "bzlib.h" : YES 
Has header "sys/soundcard.h" : YES 
Message: Checking headers needed for Open Sound System 4 plugin...
Has header "fcntl.h" : YES (cached)
Has header "sys/ioctl.h" : YES (cached)
Has header "sys/stat.h" : YES (cached)
Has header "sys/types.h" : YES (cached)
Message: Required headers found, building Open Sound System 4 plugin.
Has header "linux/videodev2.h" : YES 
Message: building v4l2 plugin
Run-time dependency gudev-1.0 found: YES 237
Run-time dependency libv4l2 found: YES 1.22.1
Run-time dependency x11 found: YES 1.7.2
Run-time dependency xext found: YES 1.3.4
Checking for function "XShmAttach" with dependency xext: YES 
Run-time dependency xfixes found: YES 6.0.0
Run-time dependency xdamage found: YES 1.1.5
Has header "aalib.h" : YES 
Library aa found: YES
Run-time dependency cairo-gobject found: YES 1.16.0
Run-time dependency flac found: YES 1.3.4
Run-time dependency gdk-pixbuf-2.0 found: YES 2.42.8
Dependency gtk+-3.0 skipped: feature gtk3 disabled
Run-time dependency jack found: YES 1.9.19
Run-time dependency libjpeg found: YES 2.1.3
Library mp3lame found: YES
Header <lame/lame.h> has symbol "lame_init" : YES 
building '/nix/store/mf6lyly23h7hidbrdy5cln93ahl9lpwl-unit-nix-gc.service.drv'...
Header <lame/lame.h> has symbol "lame_set_VBR_quality" : YES 
Header <lame/lame.h> has symbol "MEDIUM" : YES 
Run-time dependency caca found: YES 0.99.beta20
building '/nix/store/nw2n46z2w3vi8ssc1l18igh85brwlycq-options-docbook.xml.drv'...
Run-time dependency libdv found: YES 1.0.0
Run-time dependency libpng found: YES 1.6.37
Run-time dependency libmpg123 found: YES 1.29.3
Run-time dependency libraw1394 found: YES 2.1.2
Run-time dependency libavc1394 found: YES 0.5.4
Run-time dependency libiec61883 found: YES 1.2.0
Checking for function "avc1394_send_command" with dependency libavc1394: YES 
Checking for function "rom1394_free_directory" with dependency libavc1394: YES 
C++ compiler for the host machine: g++ (gcc 11.3.0 "g++ (GCC) 11.3.0")
C++ linker for the host machine: g++ ld.bfd 2.38
Run-time dependency qt5 (modules: Core, Gui, Qml, Quick) found: YES 5.15.3 (pkg-config)
Program moc-qt5 moc found: NO

ext/qt/meson.build:51:2: ERROR: Program 'moc-qt5 moc' not found or not executable

A full log can be found at /build/gst-plugins-good-1.20.1/build/meson-logs/meson-log.txt
note: keeping build directory '/tmp/nix-build-gst-plugins-good-1.20.1.drv-2'
error: builder for '/nix/store/7vbzb4vyhl97m1mbn4j3nz5q15656v95-gst-plugins-good-1.20.1.drv' failed with exit code 1
```

### Notify maintainers
@matthewbauer

### Metadata
```console
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.43, NixOS, 22.11 (Raccoon), 22.11pre-git`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.8.1`
 - channels(kn): `"nixos"`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by klemensn** [comment]
Pinging @Mindavi due to 43edcbfc7393e9a385ff2d0c5c23412742fbb483
Pinging @schmittlauch due to c4f1d21e66188faf5e16e09493e0267588f5b93c

**Comment by wozeparrot** [comment]
Pretty sure this is fixed by #176246.
